pakyow-test/lib/test_help/ext/request.rb

```diff
@@ -1,7 +1,9 @@
 module Pakyow
   class Request
-    def params
-      env['pakyow.params']
+    old_params = instance_method(:params)
+
+    define_method :params do
```

Comment:
I mean, the whole helper module is sort of a hack. That's the point of it I guess. But at least this is easier to grok.